mlflow/projects/docker.py

```diff
@@ -51,15 +73,24 @@ def validate_docker_env(project):
         )
 
 
-def build_docker_image(work_dir, repository_uri, base_image, run_id):
+def build_docker_image(work_dir, repository_uri, base_image, run_id, user_env_vars=None):
```

Comment:
Can we call this `env_vars` and add a docstring for the parameter? If there's a strong reason to call this `user_env_vars`, please let me know.

```suggestion
def build_docker_image(work_dir, repository_uri, base_image, run_id, env_vars=None):
```